projects/burnr/src/components/AccountMenu.tsx

```diff
@@ -1,22 +1,31 @@
 import React, { useState, useContext } from 'react';
 
-import { red } from '@material-ui/core/colors';
-import { Grid, Button, Typography, makeStyles, Theme, createStyles } from '@material-ui/core';
-import LanguageIcon from '@material-ui/icons/Language';
-import GetAppIcon from '@material-ui/icons/GetApp';
-import WhatshotIcon from '@material-ui/icons/Whatshot';
+import { grey } from '@material-ui/core/colors';
+import { Typography, makeStyles, Theme, createStyles, IconButton, Divider } from '@material-ui/core';
 
 import { AccountContext } from '../utils/contexts';
 
 import { openInNewTab, downloadFile, createLocalStorageAccount } from '../utils/utils';
 import { POLKA_ACCOUNT_ENDPOINTS } from '../utils/constants';
 import { useLocalStorage } from '../hooks';
 
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Menu } from '@material-ui/core';
```

Comment:
These 3 (Menu, ListItem, MenuItem) you can import them in 1 line:
import { Menu, ListItem, MenuItem } from '@material-ui/core';